src/components/AddSensorPage.vue

```diff
@@ -166,38 +197,28 @@ export default {
         required: (value) => !!value || "Required.",
         counter: (value) => value.length <= 20 || "Max 20 characters",
         devIdCounter: (value) => value.length == 24 || "Exactly 24 characters",
-        nameValidator: (value) => {
-          // First char can be a (non)capital letter, all other chars can only be non-capital letters!
-          const pattern = /^([a-zA-Z][a-z]+([ ]?[[a-zA-Z][a-z]+)*)$/;
-          return (
-            pattern.test(value) ||
-            "Invalid Name: Can only be 1 (non)capital letter + lowercase letters."
-          );
-        },
-        lastnameValidator: (value) => {
-          // First char can be a (non)capital letter, all other chars can only be non-capital letters, 1 or more names!
-          const pattern = /^([a-zA-Z][a-z]+([ ]?[a-z]?['-]?[a-zA-Z][a-z]+)*)$/;
-          return (
-            pattern.test(value) ||
-            "Invalid Name: Can only be 1 (non)capital letter + lowercase letters (- and ')."
-          );
-        },
         deviceidValidator: (value) => {
           // Can only be a hexadecimal value
           const pattern = /^[a-fA-F0-9]+$/;
-          return (
-            pattern.test(value) ||
-            "Invalid Device ID: Can only be a hexadecimal value. (Letters from a to f, and numbers)"
-          );
+          return pattern.test(value) || this.deviceidText;
         },
         devicenameValidator: (value) => {
           // Can only be lowercase letters, numbers, underscores or dashes (multiple words)
-          const pattern = /^([a-zA-Z-_]+([ ]?[a-z]?['-]?[a-zA-Z-_]+)*)$/;
+          const pattern = /^([a-zA-Z-_]+([a-z]?['-]?[a-zA-Z-_]+)*)$/;
 
-          return (
-            pattern.test(value) ||
-            "Invalid Device Name: Can only be letters, numbers, underscores or dashes."
-          );
+          return pattern.test(value) || this.devicenameText;
+        },
+        latValidator: (value) => {
+          const pattern =
+            /^(\+|-)?(?:90(?:(?:\.0{1,6})?)|(?:[0-9]|[1-8][0-9])(?:(?:\.[0-9]{1,6})?))$/;
+
+          return pattern.test(value) || this.devicelocationText;
+        },
+        longValidator: (value) => {
+          const pattern =
+            /^(\+|-)?(?:180(?:(?:\.0{1,6})?)|(?:[0-9]|[1-9][0-9]|1[0-7][0-9])(?:(?:\.[0-9]{1,6})?))$/;
```

Comment:
Length of 6 is not enough, google maps default is 7 decimal places `50.8461056` `3.3062912`